t04_refactored.py

```diff
@@ -182,7 +183,105 @@ def team_11_adv():
 
 ###################################################################################
 
+
 def team_12_adv():
+    """https://docs.google.com/document/d/13xlcbGqW_CVr6wpsi9CH2nEjf8Qk6_S0cyVs3417i_I/edit?usp=sharing
+    Baella Morgan, Ariana Meatchem, Jaren Vessels"""
+    global HasSword
+    global goblindead
+    # Intro
+    print("You stand in your makeshift campsite, a pitiful fire smoulders, sending black smoke to the sky.")
+    sleep(delay)
+    print("To the North: The Mystic Wood lies, fabeled to hold a great treasure.")
+    sleep(delay)
+    print("To the South: The Mouth of a cave, you can hear noises creep from within.")
+    sleep(delay)
+    print("To the West: A long road that you came from.")
+    sleep(delay)
+    print("To the East: A vast meadow of yellow flowers.")
+    sleep(delay)
+    ############
+    direction = input("Choose a direction or action[Search the camp/North/South/East/West]: ")
+    # CHECK FOR VALID INPUT###
+    while direction != "north" and direction != "south" and direction != "east" and direction != "west" and direction != "search camp":
+        print("Now", username, " That wasn't a valid option.")
+        direction = input("Choose a direction or action[Search the camp/North/South/East/West]: ")
+
+    # Print string based on input###
+    if direction == "north":
+
+        if not goblindead:
+            print("You go north.")
+            sleep(delay)
+            print("The trees tower over you almost nearly blocking out the sun.")
+            sleep(delay)
+            print("Suddenly a large goblin bandit jumps from the bushes roaring")
+            sleep(delay * 3)
+            # check if user has sword
+        if HasSword:
+            print("The goblin lunges at you, your swords clash.")
+            sleep(delay)
+            print("You best the goblin, stabbing him in the chest.")
+            print("You step over the goblins corpse, procceding deeper into the forest.")
+            goblindead = True
+            sleep(delay * 3)
+            room2()
+
+        elif HasSword:
```

Comment:
Because of the previous two comments, this code is unreachable (i.e., the dead code warning in PyCharm).